interfaces/fiveg_n2/v0/README.md

```diff
@@ -0,0 +1,46 @@
+# `fiveg_n2`
+
+## Usage
+
+Within 5G, The Access and Mobility Management Function (AMF) is responsible for managing access and mobility for 5G devices.
```

Comment:
Here you introduce the AMF and then you talk about N2 but never connect the dots.

Reply:
I added the explanation we have in the spec.
I was avoiding it just not to be very specific naming 5G components, but N2 seems to be pretty specific anyway.
Let me know if it looks better now.